src/box/lua/tuple.c

```diff
@@ -713,6 +720,92 @@ static const struct luaL_Reg lbox_tuple_iterator_meta[] = {
 	{NULL, NULL}
 };
 
+struct tuple_format *
+lbox_check_tuple_format(struct lua_State *L, int narg)
```

Comment:
Is there a reason to don't move all the format related Lua functions into its own file: say, src/box/lua/tuple_format.c?

As for me, the lua/tuple module is already quite complicated.

Reply:
I think code related to one Lua module should be located in one file.

Reply:
But what is the purpose? It just makes unrelated functions and initialization code resides in one file. It doesn't help with reading/understanding it anyhow.

The `box.tuple.format` is more like a module/class, just exposed as part of the box.

I don't insist, but I will leave this discussion open. If no other has an opinion on it, let's choose what you personally prefer.

If anyone other has a preference here, let's share it.

Reply:
Moved this code into a new `tuple_format` submodule.